### DISCUSSION TITLE
본문 글 올린 뒤 대표 이미지 상단 미 적용 부분

### DISCUSSION THREAD

**Discussion by Bonguk** [pronist/hELLO#116] [Q&A] [answered]
**환경 (버그가 발생한 환경을 적어주세요.):**

- 운영체제: Windows 11
- 브라우저: chrome 103.0.5060.53(공식 빌드) (64비트)
- 버전: [e.g. 22]
- 스킨 버전 (스킨 버전은 index.xml 에서 확인): 3.6.1

**설명**
대표 이미지를 등록 했음에도 본문 글 상단 부분 이미지 미 적용
이미지 확장자 : png
이미지 크기 : 600 x 200 

**발생 과정**
본문 글 작성 후 대표 이미지 설정 했을 때

**스크린샷 및 링크**
![image](https://user-images.githubusercontent.com/47359380/176206266-e59d1fa2-da27-4018-880c-f6c5814f7bcb.png)

**Comment by pronist** [comment, marked as answer]
@Bonguk 안녕하세요. 이 부분은 **스킨 옵션**에서 **글 읽기 모드**를 변경해주시기 바랍니다.
